src/open_inwoner/configurations/management/commands/generate_config_docs.py

```diff
@@ -0,0 +1,115 @@
+import os
+from pathlib import Path
+from typing import TypeAlias
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from jinja2 import Environment, FileSystemLoader, select_autoescape
+
+from open_inwoner.configurations.bootstrap.models import (
+    DigiDOIDCConfigurationSettings,
+    KICConfigurationSettings,
+    SiteConfigurationSettings,
+    ZGWConfigurationSettings,
+    eHerkenningDOIDCConfigurationSettings,
+)
+
+ConfigSetting: TypeAlias = (
+    DigiDOIDCConfigurationSettings
+    | KICConfigurationSettings
+    | SiteConfigurationSettings
+    | ZGWConfigurationSettings
+    | eHerkenningDOIDCConfigurationSettings
+)
+
+
+SUPPORTED_OPTIONS = ["siteconfig", "kic", "zgw", "digid_oidc", "eherkenning_oidc"]
+
+TEMPLATE_DIR = (
+    Path(os.path.abspath(os.path.dirname(__file__))).parent.parent
+    / "bootstrap"
+    / "templates"
+)
+TARGET_DIR = Path(settings.BASE_DIR) / "docs" / "configuration"
+
+
+class Command(BaseCommand):
+    help = "Create docs for configuration setup steps"
+
+    def add_arguments(self, parser):
+        parser.add_argument("config_option", nargs="?")
+
+    def get_config(self, config_option: str) -> dict[str, ConfigSetting]:
+        mapping = {
+            "siteconfig": SiteConfigurationSettings,
+            "kic": KICConfigurationSettings,
+            "zgw": ZGWConfigurationSettings,
+            "digid_oidc": DigiDOIDCConfigurationSettings,
+            "eherkenning_oidc": eHerkenningDOIDCConfigurationSettings,
+        }
+        return mapping[config_option]
+
+    def get_detailed_info(self, config: ConfigSetting) -> list[str]:
+        ret = []
+        for field in config.fields["all"]:
+            model_field = config.model._meta.get_field(field.name)
+            part = []
+            part.append(f"{'Variable':<20}{config.get_setting_name(field)}")
+            part.append(f"{'Setting':<20}{str(model_field.verbose_name)}")
+            part.append(f"{'Description':<20}{str(model_field.help_text)}")
+            part.append(f"{'Model field type':<20}{field.field_type}")
+            part.append(f"{'Possible values':<20}{field.values}")
+            part.append(f"{'Default value':<20}{field.default_value}")
+            ret.append(part)
+        return ret
+
+    def write_file_from_template(
+        self,
+        template_path: os.PathLike,
+        template_variables: dict[str, list],
+        output_path: os.PathLike,
+    ):
+        with open(template_path, "r") as template:
+            template_str = template.read()
+            template = Environment(
+                loader=FileSystemLoader(TEMPLATE_DIR), autoescape=select_autoescape()
+            ).from_string(template_str)
```

Comment:
We should stick to one template engine, in this case Django templates, and follow the regular template finder/loader pattern, instead of introducing custom Jinja that is not used elsewhere and afaics not specifically necessary here. 

So move the templates to a normal Django app template directory, with a subdirectory named 'configurations' (the app name) and a normal .rst extension (without .template). They are very specific to this app so you can keep the templates in the app directory (and not the shared templates in the project root).

Reply:
I couldn't find a way to get rid of spacing between the settings (they are all separted by a newline when generated). `{% spaceless %}` doesn't seem to work, and Jinja has a convenient method to trim blocks of content (e.g. `{%- content %}` will eliminate newlines before `content`). Is there a way to achieve this with Django templates?